spec/mongoid/interceptable_spec.rb

```diff
@@ -170,6 +170,65 @@ class TestClass
     it "runs after document instantiation" do
       expect(game.name).to eq("Testing")
     end
+
+    context 'when the document is embedded' do
+
+      before do
+        Book.destroy_all
+      end
+
+      after do
+        Book.destroy_all
+      end
+
+      let(:book) do
```

Comment:
The spacing was leftover from copy-and-pasting from the ticket info. I've updated it.